src/js/components/Notification/README.md

```diff
@@ -0,0 +1,14 @@
+## Notification
+Is a container for notifications
+
+[![](https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png)](https://storybook.grommet.io/?selectedKind=Controls-Notification&full=0&stories=1&panelRight=0) [![](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/grommet/grommet-sandbox?initialpath=/notification&module=%2Fsrc%2FNotification.js)
+## Usage
+
+```javascript
+import { Notification } from 'grommet';
+      <Notification />
+```
+
+## Properties
```

Comment:
Take a look at the Readme format for other components, the readme should just include a link to that component's documentation